packages/backend/src/clients/Slack/SlackClient.ts

```diff
@@ -11,6 +11,9 @@ type SlackClientDependencies = {
     lightdashConfig: LightdashConfig;
 };
 
+const CACHE_TIME = 1000 * 10 * 60; // 10 minutes
+let cachedChannels: { lastCached: Date; channels: SlackChannel[] } | undefined;
```

Comment:
This won't work because we need to cache the results for multiple slack integrations.

Reply:
Already mentioned that, will fix https://github.com/lightdash/lightdash/pull/4554#issuecomment-1439742752